src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -104,9 +108,55 @@ def __call__(self, organization, event):
                     pass
 
 
+class PullEventWebhook(Webhook):
+    def __call__(self, organization, event):
+        repo = self.repo_from_event(organization, event)
+        pull_request = event['pullrequest']
+
+        integration = Integration.objects.get(
+            id=repo.integration_id,
+        )
+        installation = integration.get_installation(organization_id=organization.id)
+        emails = installation.get_client().get_user_emails()['values']
+        try:
+            commit_author = CommitAuthor.objects.filter(
+                organization_id=organization.id,
+                email__in=[email['email'] for email in emails if email['is_confirmed']],
+            )[0]
+        except IndexError:
+            primary_email = None
+            for email in emails:
+                if email['is_primary']:
+                    primary_email = email['email']
+                    break
```

Comment:
Is one of the emails always guaranteed to be `primary`?

Reply:
@mnoble Hmm so according to bitbcuket 1.0 it would appear so.https://confluence.atlassian.com/bitbucket/emails-resource-296911737.html But bitbucket 2.0 doesn't address emails in depth in the documentation. I'm inclined to think that it does. But, if there is no primary email like you're wondering how do you decide which email to use?